README.md

```diff
@@ -122,6 +122,28 @@ Whether you're using a physical device or simulator, you should now see a Hyperv
 
 The example server simply serves files from the [./examples](/examples) directory. You can modify or add files in [./examples](/examples) and the server will update without restarting.
 
+#### Running storybook
+
+1. Run the following to generate storybook:
+
+```bash
+yarn generate applicant
```

Comment:
this is specific to our internal app, I think we should explain here that `applicant` is the name of the react-native root component as specified in the `AppRegistry.registerComponent` method call (see https://github.com/Instawork/hyperview/blob/master/scripts/generate/app-key.js#L20)